src/components/NavTop.js

```diff
@@ -29,30 +13,38 @@ const StyledH2 = styled.h2`
   font-style: none;
 `;
 
-const LeaveSettings = styled(Cross)`
-  height: 35px;
-  width: 35px;
+const SettingsInput = styled.button`
+  background: ${props => props.theme.contrast};
+  color: ${props => props.theme.brightFont};
+  fill: ${props => props.theme.brightFont};
+  border: ${props => props.theme.contrast};
+  width: 40px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-items: center;
+  border-radius: 50%;
+  margin: 0;
   padding: 0;
-  min-width: 0;
 `;
 
 function NavTop({ state, round, onClick }) {
   if (state === "before") {
     return (
       <StyledHeader>
-        <RoundButton onClick={onClick} round={round}>
-          <CheckSettings></CheckSettings>
-        </RoundButton>
+        <SettingsInput onClick={onClick} round={round}>
+          <Cross></Cross>
```

Comment:
Maybe it should be called `CrossIcon` instead of `Cross`?

That also applies to the other icons.

Reply:
I know what you mean. I took "around spaces" as an example. There we did not include the components "category" into their name. Therefore I will leave it as is. But thx for the Feedback!